app/scripts/controllers/transactions/index.js

```diff
@@ -362,6 +362,9 @@ export default class TransactionController extends EventEmitter {
       type: TRANSACTION_TYPE_CANCEL,
     })
 
+    const { transactionCategory } = await this._determineTransactionCategory(txParams)
```

Comment:
Adding the transaction category to speedups makes a lot of sense, but I'm not sure it's useful for cancellations 🤔 The transaction category should accompany a transaction with all of the metadata expected for that category of transaction, and that just isn't the case for cancellations.

Can you think of a place where this would be useful? Maybe we can add the category just to speedups for now otherwise.